lib/decrypt.rb

```diff
@@ -0,0 +1,60 @@
+require_relative 'file_input_output'
+require_relative 'alphabet_hash'
+
+class Decrypt
+  include AlphabetHash
+  attr_reader :alphabet,
+              :line1,
+              :line2,
+              :line3,
+              :line1_braille,
+              :line2_braille,
+              :line3_braille,
+              :braille_letters,
+              :file_input_output
+
+  def initialize
+    @alphabet = AlphabetHash.alphabet_hash.invert
+    @file_input_output = FileInputOutput.new
+    @line1 = []
+    @line2 = []
+    @line3 = []
+    @line1_braille = []
+    @line2_braille = []
+    @line3_braille = []
+    @braille_letters = []
+  end
+
+  def decrypt_file
+    braille = @file_input_output.read.chomp
+    english = decrypt_to_english(braille)
+    @file_input_output.write(english)
```

Comment:
29 - 31 are not being tested. they are runner methods however. so i don't know that I need to.